storage/immutable_segment.go

```diff
@@ -207,6 +204,9 @@ func (s *immutableSeg) QueryRaw(
 		}
 	}()
 
+	if queryFields[1] == nil {
```

Comment:
make this a const? `rawDocQueryFieldIdx` or something

Reply:
sounds good